apps/builder/app/builder/features/breakpoints/breakpoints-editor.tsx

```diff
@@ -73,7 +73,13 @@ const BreakpointEditorItem = ({
         <TextField
           css={{ textAlign: "right", width: 50 }}
           variant="ghost"
-          defaultValue={breakpoint.minWidth}
+          defaultValue={
+            "minWidth" in breakpoint
+              ? breakpoint.minWidth
+              : "maxWidth" in breakpoint
+              ? breakpoint.maxWidth
+              : 0
+          }
```

Comment:
`breakpoint.minWidth ?? breakpoint.maxWidth ?? 0`

Reply:
right I had a different type before, it hat do check like I did before, but now it can be simplified